cypress/integration/common/utils.ts

```diff
@@ -192,13 +192,15 @@ export class Utilities {
   }
 
   click(x: number, y: number): void {
-    this.getScrollableElement().trigger("pointerdown", x, y, {
-      pointerType: "mouse",
-    });
-    this.getBody().trigger("pointerup", 0, 0, {
-      pointerType: "mouse",
-      force: true,
-    }); //
+    // TODO: use pointerdown + pointerup instead of click
```

Comment:
Why was this changed? Short comment would be helpful.

Reply:
I think that it's related to cypress:
https://github.com/cypress-io/cypress/issues/21381

I didn't see that we import cypress/react so I did nothing about it but I will add a comment.